client/src/margo_client.c

```diff
@@ -273,8 +283,18 @@ int invoke_client_unmount_rpc(void)
     return (int)ret;
 }
 
-/* invokes the client metaset rpc function */
-int invoke_client_metaset_rpc(unifyfs_file_attr_t* f_meta)
+/*
+ * Set the metadata values for a file (after optionally creating it).
+ * The gfid for the file is in f_meta->gfid.
+ *
+ * create:If set to 1, attempt to create the file first.  If the file
+ *        already exists, then update its metadata with the values in
+ *        f_meta.  If set to 0, and the file does not exist, then
```

Comment:
`If set to 0, and the file does not exist, then the server will return an error.` - I can't remember, is this the current behavior with this patch?  If not, then we should remove that sentence.

Reply:
that's still true